client/post-editor/editor-drawer/label.jsx

```diff
@@ -3,16 +3,25 @@
  */
 import React, { PropTypes } from 'react';
 
-export default function EditorDrawerLabel( { children } ) {
+/**
+ * Internal dependencies
+ */
+import InfoPopover from 'components/info-popover';
+
+export default function EditorDrawerLabel( { children, labelText, helpText } ) {
 	return (
 		<label className="editor-drawer__label">
 			<span className="editor-drawer__label-text">
+				{ labelText }
+				{ helpText && <InfoPopover position="top left">{ helpText }</InfoPopover> }
 				{ children }
```

Comment:
Took this for a spin and while it appears to display correctly, it does so only because the input fields often override the label text styling. Rather, I think `children` should be moved outside the `span` here to avoid those styles being applied altogether.